src/views/Samples/View.tsx

```diff
@@ -0,0 +1,33 @@
+import { SAMPLES_PER_PAGE } from "@temp/core/config";
+import * as React from "react";
+import { RouteComponentProps } from "react-router-dom";
+import Page from "./Page";
+import { TypedAllSampleQuery } from "./queries";
+// import './scss/index.scss';
+
+type ViewProps = RouteComponentProps<{ id: string }>;
+
+const STATIC_CAT_SAMPLE_ID = "Q2F0ZWdvcnk6MjM=";
```

Comment:
Store as TS enum in `src/constants.ts` as this is reused.
https://www.typescriptlang.org/docs/handbook/enums.html
```
export enum SaleorCategoryIds {
  Sample = "Q2F0ZWdvcnk6MjM=",
}
```

Also remember to change in `src/views/CollectionIndex/Samples/View.tsx`

Reply:
Done